src/models/Call.ts

```diff
@@ -785,6 +788,16 @@ export class ElementCall extends Call {
         audioInput: MediaDeviceInfo | null,
         videoInput: MediaDeviceInfo | null,
     ): Promise<void> {
+        this.kickedOutByAnotherDevice = false;
+        this.client.on(ClientEvent.ToDeviceEvent, this.onToDeviceEvent);
+
+        this.connectionTime = Date.now();
+        await this.client.sendToDevice(ElementCall.DUPLICATE_CALL_DEVICE_EVENT_TYPE, {
+            [this.client.getUserId()]: {
+                "*": { device_id: this.client.getDeviceId(), timestamp: this.connectionTime },
+            },
+        });
```

Comment:
Is removing other devices from the `m.call.member` state event not sufficient to kick them out? (It'd be nice if we could do everything with plain MSC3401 events)

Reply:
I've had a go at this but couldn't get it working and I am currently just lacking the time to figure out what exactly is going on. Since we're being pushed by time, unless someone else wants to take this over, I would suggest merging and putting this into our backlog

Sorry about all of this, hopefully, the situation should improve during the next week...

Reply:
OK, that's fine then.